### DISCUSSION TITLE
Arabic

### DISCUSSION THREAD

**Discussion by bsyy6** [monkeytypegame/monkeytype#3491] [Ideas] [answered]
The arabic language text has all the accents on top:
1) In arabic we don't usually type them, that can be set as hard option or a second option for Arabic.
2) If I ignore them, I have to type each letter twice for the cursor to pass to the next character making it impossible to type fast.
current version
![image](https://user-images.githubusercontent.com/28226591/188198608-793c959a-57f8-462a-9322-abe17e536ad5.png)
suggested:
![image](https://user-images.githubusercontent.com/28226591/188198727-336a4422-728f-439d-a743-8ae3912df46f.png)

**Comment by Miodec** [comment, marked as answer]
You can enable lazy mode to remove accents
